### PR TITLE
A New Order article crew

### PR DESCRIPTION
**Pull request by skitau**

### CODE REVIEW

File: data/upgrades/crew.json [hunk 1]
```diff
@@ -198,11 +198,12 @@
       {
         "title": "Captain Phasma",
         "type": "Crew",
-        "ability": "At the...",
+        "ability": "At the end of the Engagement Phase, each enemy ship at range 0-1 that is not stressed gains 1 stress token.",
         "slots": ["Crew"]
       }
     ],
     "cost": { "value": 0 }
```
Comment (guidokessels) [inline data/upgrades/crew.json]: There's a missing `,` at the end of this line.

File: data/upgrades/crew.json [hunk 2]
```diff
@@ -617,6 +632,22 @@
...
+    "name": "Kylo Ren",
+    "limited": 1,
+    "xws": "kyloren",
+    "sides": [
+      {
+        "title": "Kylo Ren",
+        "type": "Crew",
+        "ability": "Action: Choose 1 enemy ship at range 1-3. If you do, spend 1 [Force] to assign the I'll Show You the Dark Side condition to that ship.",
+        "conditions": ["illshowyouthedarkside"],
+        "slots": ["Crew"]
```
Comment (guidokessels) [inline data/upgrades/crew.json]: Same here; Missing a `,` at the end of this line.

File: data/upgrades/crew.json [hunk 3]
```diff
@@ -975,11 +1020,13 @@
...
         "title": "Supreme Leader Snoke",
         "type": "Crew",
-        "ability": "During the System... you may...",
-        "slots": ["Crew", "Crew"]
+        "ability": "During the System Phase, you may choose any number of enemy ships beyond range 1. If you do, spend that many [Force] to flip each chosen ship's dial faceup.",
+        "slots": ["Crew", "Crew"],
+        "force": { "value": 1, "recovers": 1 },
       }
     ],
     "cost": { "value": 0 }
```
Comment (guidokessels) [inline data/upgrades/crew.json]: Same here: Missing `,` at the end of this line.

**Comment by guidokessels** [comment]
Thanks! I left a few comments.

**Comment by skitau** [comment]
I thought I had all the end of lines sorted, sorry!

**Comment by guidokessels** [comment]
I've fixed the commas in a new PR and merged that one: https://github.com/guidokessels/xwing-data2/pull/101

Thanks!

**Pull request closed without merging**